qdrant-landing/content/documentation/data-ingestion-beginners.md

```diff
@@ -235,13 +172,13 @@ In Qdrant, data is organized in collections, each representing a set of embeddin
 
 Here’s how to create a collection in Qdrant to store both text and image embeddings:
 
-```jsx
+```python
 def create_collection(collection_name):
     qdrant_client.create_collection(
         collection_name,
         vectors_config={
             "text_embedding": models.VectorParams(
-                size=1536,  # Dimension of text embeddings
+                size=384,  # Dimension of text embeddings
```

Comment:
Code explanation coming right after the code snippet still refers to text embeddings being 1536 instead of 384